regen/regcomp.pl

```diff
@@ -17,6 +18,34 @@
 # Accepts the standard regen_lib -q and -v args.
 #
 # This script is normally invoked from regen.pl.
+#
+# F<regcomp.sym> defines the opcodes and states used in the regex
+# engine, it also includes documentation on the opcodes. This script
+# parses those definitions out and turns them into typedefs, defines,
+# and data structures, and maybe even code which the regex engine can
+# use to operate.
+#
+# F<regexp.h> and op_reg_common.h contain defines C<RXf_xxx> and
+# C<PREGf_xxx> that are used in flags in our code. These defines are
+# parsed out and data structures are created to allow the debug mode of
+# the regex engine to show things like which flags were set during
+# compilation. In some cases we transform the C code in the header files
+# into perl code which we execute to C<eval()> the contents. For instance
+# in a situation like this:
+#
+#   #define RXf_X 0x1   /* the X mode */
+#   #define RXf_Y 0x2   /* the Y mode */
+#   #define RXf_Z (X|Y) /* the Z mode */
+#
+# this script might end up eval()ing something like C<0x1> and then
```

Comment:
s/^this script/This script/

Reply:
I disagree, `this script` is not beginning a sentence

Reply:
> I disagree, `this script` is not beginning a sentence

Well, then, perhaps we could place ellipses at the end of the phrase above the code sample and at the beginning of the phrase after the code sample.  (This is a suggestion, not a requirement.)

Reply:
@khwilliamson, are you okay with my revised suggestion above?